C/cryptography/Affine_Substitution_Cipher.c

```diff
@@ -0,0 +1,143 @@
+// C program to illustate Affine Cipher
+
+#include<stdio.h>
+#include<string.h>
+#include<ctype.h>
+#include<stdlib.h>
+// function declaration
+int CalculateGCD(int);
+int CalculateMultiplicativeInverse(int);
+
+int main() {
+  int index, indx, k, gcd, alpha, beta, num_msg[100], numcipher[100], numdecipher[100], decipher[100], alphaInverse;
+  char msg[100], cipher[100];
```

Comment:
Don't restrict your cipher for 100 lengths make it more scalable.

Take the size dynamically

Reply:
Updated the file. Kindly review @plazzy99